File: Domain/Types/Slide.cs
namespace Domain.Types
{
    using System;

    public class Slide
    {
        public long Id { get; set; }

        public string Provider { get; set; }

        public string ReferenceNumber { get; set; }

        public string SlideType { get; set; }

        public DateTime? From { get; set; }

        public DateTime? To { get; set; }
    }
}

File: KnockoutDemo/Models/Slide.js
(function(knockoutDemo) {

    // Slide constructor function
    function Slide() {

        // Get the reference to self
        var self = this;

        // Define properties
        self.Provider = window.ko.observable("");
        self.ReferenceNumber = window.ko.observable("");
        self.SlideType = window.ko.observable("");
        self.From = window.ko.observable("");
        self.To = window.ko.observable("");

    }

    // Add to the namespace
    knockoutDemo.Slide = Slide;

} (window.knockoutDemo));